webapi/Lib/WebHooks/MccSoft.WebHooks/Manager/WebHookManager.cs

```diff
@@ -64,10 +69,20 @@ public async Task<TSub> Subscribe(
         var subscription = (TSub)
             Activator.CreateInstance(typeof(TSub), [name, url, eventType, method, headers]);
 
+        // TODO: think how to expose it from here
+        var secret = _signatureService.GenerateEncryptedSecret(subscription);
+
         _dbContext.WebHookSubscriptions<TSub>().Add(subscription);
         await _dbContext.SaveChangesAsync();
 
-        return subscription;
+        // temporary solution for providing insecure secret to user just after sub creation.
```

Comment:
I guess the comment and `.UpdateSignatureSecret` call should be removed?

Reply:
yes - forgot to adjust it